apps/src/applab/designMode.js

```diff
@@ -473,7 +474,8 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
   if (!Applab.levelHtml) {
     return;
   }
-  var levelDom = $.parseHTML(Applab.levelHtml);
+  var sanitized = sanitizeHtml(Applab.levelHtml);
```

Comment:
Do we not get any feedback if Applab.levelHtml sanitizes away some contents? I think we'd discussed perhaps just failing in that case? Seems reasonable if the plan is just to do that in a future commit so that these changes are ready for tomorrow.